.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
         run: echo "::set-env name=CARGO_HOME::$(pwd)/target/cargo"
 
       - name: Cache
-        uses: actions/cache@master
+        uses: denoland/github-actions-cache@stable-prerelease
```

Comment:
Why do you need to fork this?

Reply:
Because actions/cache@master is broken and the most recent tagged version doesn't support caching multiple directories.